components/automate-cli/cmd/chef-automate/automateHAAwsDeployment.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"github.com/chef/automate/components/automate-cli/pkg/status"
+)
+
+type awsDeployment struct{}
+
+func newAwsDeployemnt() *awsDeployment {
+	return &awsDeployment{}
+}
+
+func (a *awsDeployment) doDeployWork(args []string) error {
+	if isA2HARBFileExist() {
```

Comment:
why do we need to check this again, if we had done this check when we did it in getDeployer->getModeFromConfig->checkIfFileExist